test/Libraries/CoreNodesTests/CoreNodeModelsTests/BaseTypesTests.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using CoreNodeModels.Input;
+using Dynamo;
+using Dynamo.Tests;
+using NUnit.Framework;
+using ProtoCore.AST.AssociativeAST;
+
+namespace DSCoreNodesTests
+{
+    [TestFixture]
+    public class SequenceTests : DynamoModelTestBase
+    {
+        private DoubleInput DoubleInputNode;
+        private DoubleInput.IDoubleSequence testNode;
+
+        [SetUp]
+        public void TestsSetup()
+        {
+            string openPath = Path.Combine(TestDirectory,
+                @"core\number\TestNumberDeserialization.dyn");
+            OpenModel(openPath);
+
+            DoubleInputNode = CurrentDynamoModel.CurrentWorkspace.NodeFromWorkspace<DoubleInput>(
+                Guid.Parse("6fc905f8533f433a90fe4b9181463d53"));
+
+            testNode = DoubleInput.ParseValue("1:#TestText", new[] { '\n' }, new List<string>(), DoubleInputNode.Convert).FirstOrDefault();
+        }
+
+        [Test]
+        public void GetFSchemeValueTest()
+        {
+            Dictionary<string, double> idLookup = new Dictionary<string, double>();
+            idLookup.Add("TestText", 1);
+
+            IEnumerable<double> result = (IEnumerable<double>) testNode.GetFSchemeValue(idLookup);
```

Comment:
I don't think `GetFSchemeValue` is used any longer. It is from the old days when the Dynamo language was based on FScheme.

Reply:
Please don't add a test for this method and mark it obsolete.

Reply:
ok I will update that on the next commit